azurelinuxagent/ga/collect_logs.py

```diff
@@ -169,18 +169,12 @@ def _collect_logs(self):
 
         systemd_cmd = [
             "systemd-run", "--unit={0}".format(logcollector.CGROUPS_UNIT),
-            "--slice={0}".format(cgroupconfigurator.AZURE_SLICE)
+            "--slice={0}".format(cgroupconfigurator.LOGCOLLECTOR_SLICE), "--scope"
```

Comment:
Just to make sure, are we managing logcollector under `azure.slice/azure-walinuxagent-periodic_logcollector.slice/collect-logs.scope` but what I read is systemd-run creates the scope under the system slice by default as per our implementation for extension slice
https://github.com/Azure/WALinuxAgent/blob/develop/azurelinuxagent/common/cgroupapi.py#L258. 
May be either defining slice name in contents or something like creating file under azure.slice should do the trick.

Reply:
As per our offline sync, `systemd-run` is unfortunately a bit wild with how it generates the hierarchical slices: we're passing `azure-walinuxagent-periodic_logcollector.slice` as the slice name here, and `systemd-run` translates that to `azure.slice/azure-walinuxagent.slice/azure-walinuxagent-periodic_logcollector.slice` by using the `-`s in the original as delimiters.